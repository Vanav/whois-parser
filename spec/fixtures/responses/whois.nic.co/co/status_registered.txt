Domain Name: t.co
Registry Domain ID: D740225-CO
Registrar WHOIS Server: whois.corporatedomains.com
Registrar URL: whois.corporatedomains.com
Updated Date: 2017-04-21T05:57:23Z
Creation Date: 2010-04-26T07:50:40Z
Registry Expiry Date: 2018-04-25T23:59:59Z
Registrar: CSC Corporate Domains
Registrar IANA ID: 299
Registrar Abuse Contact Email: domainabuse@cscglobal.com
Registrar Abuse Contact Phone: +1.8887802723
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: serverTransferProhibited https://icann.org/epp#serverTransferProhibited
Domain Status: serverDeleteProhibited https://icann.org/epp#serverDeleteProhibited
Domain Status: serverUpdateProhibited https://icann.org/epp#serverUpdateProhibited
Registry Registrant ID: C49734528-CO
Registrant Name: Twitter, Inc.
Registrant Organization: Twitter, Inc.
Registrant Street: 1355 Market Street
Registrant Street: Suite 900
Registrant Street:
Registrant City: San Francisco
Registrant State/Province: CA
Registrant Postal Code: 94103
Registrant Country: US
Registrant Phone: +1.4152229670
Registrant Phone Ext:
Registrant Fax: +1.4152220922
Registrant Fax Ext:
Registrant Email: domains@twitter.com
Registry Admin ID: C49734526-CO
Admin Name: Domain Admin
Admin Organization: Twitter, Inc.
Admin Street: 1355 Market Street
Admin Street: Suite 900
Admin Street:
Admin City: San Francisco
Admin State/Province: California
Admin Postal Code: 94103
Admin Country: US
Admin Phone: +1.4152229670
Admin Phone Ext:
Admin Fax: +1.4152220922
Admin Fax Ext:
Admin Email: domains@twitter.com
Registry Tech ID: C49734527-CO
Tech Name: Tech Admin
Tech Organization: Twitter, Inc.
Tech Street: 1355 Market Street
Tech Street: Suite 900
Tech Street:
Tech City: San Francisco
Tech State/Province: California
Tech Postal Code: 94103
Tech Country: US
Tech Phone: +1.4152229670
Tech Phone Ext:
Tech Fax: +1.4152220922
Tech Fax Ext:
Tech Email: domains-tech@twitter.com
Name Server: d01-01.ns.twtrdns.net
Name Server: d01-02.ns.twtrdns.net
Name Server: r01-02.ns.twtrdns.net
Name Server: ns4.p34.dynect.net
Name Server: ns3.p34.dynect.net
Name Server: r01-01.ns.twtrdns.net
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2018-01-12T18:15:18Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

.CO Internet, S.A.S., the Administrator for .CO, has collected this information for the WHOIS database through Accredited Registrars. This information is provided to you for informational purposes only and is designed to assist persons in determining contents of a domain name registration record in the .CO Internet registry database. .CO Internet makes this information available to you "as is" and does not guarantee its accuracy.

By submitting a WHOIS query, you agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data:  (1) to allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via direct mail, electronic mail, or by telephone; (2) in contravention of any applicable data and privacy protection laws; or (3) to enable high volume, automated,  electronic processes that apply to the registry (or its systems). Compilation, repackaging, dissemination, or other use of the WHOIS database in its entirety, or of a substantial portion thereof, is not allowed without .CO Internet's prior written permission. .CO Internet reserves the right to modify or change these conditions at any time without prior or subsequent notification of any kind. By executing this query, in any manner whatsoever, you agree to abide by these terms.  In some limited cases, domains that might appear as available in whois might not actually be available as they could be already registered and the whois not yet updated and/or they could be part of the Restricted list. In this cases, performing a check through your Registrar's (EPP check) will give you the actual status of the domain. Additionally, domains currently or previously used as extensions in 3rd level domains will not be available for registration in the 2nd level. For example, org.co,mil.co,edu.co,com.co,net.co,nom.co,arts.co, firm.co,info.co,int.co,web.co,rec.co,co.co.

NOTE: FAILURE TO LOCATE A RECORD IN THE WHOIS DATABASE IS NOT INDICATIVE OF THE AVAILABILITY OF A DOMAIN NAME. All domain names are subject to certain additional domain name registration rules. For details, please visit our site at www.cointernet.co <http://www.cointernet.co>.
