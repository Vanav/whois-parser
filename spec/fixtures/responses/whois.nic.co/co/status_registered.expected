#domain
  %s == "t.co"

#domain_id
  %s == "D740225-CO"


#status
  %s == ["clientTransferProhibited"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2010-04-26 07:50:40 GMT}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-04-21 05:57:23 GMT}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-04-25 23:59:59 GMT}


#registrar
  %s %CLASS{registrar}
  %s.id           == "299"
  %s.name         == "CSC Corporate Domains"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "C49734528-CO"
  %s[0].name          == "Twitter, Inc."
  %s[0].organization  == "Twitter, Inc."
  %s[0].address       == "1355 Market Street\nSuite 900"
  %s[0].city          == "San Francisco"
  %s[0].zip           == "94103"
  %s[0].state         == "CA"
  %s[0].country       == "United States"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4152229670"
  %s[0].fax           == "+1.4152220922"
  %s[0].email         == "domains@twitter.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "C49734526-CO"
  %s[0].name          == "Domain Admin"
  %s[0].organization  == "Twitter, Inc."
  %s[0].address       == "1355 Market Street\nSuite 900"
  %s[0].city          == "San Francisco"
  %s[0].zip           == "94103"
  %s[0].state         == "California"
  %s[0].country       == "United States"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4152229670"
  %s[0].fax           == "+1.4152220922"
  %s[0].email         == "domains@twitter.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "C49734527-CO"
  %s[0].name          == "Tech Admin"
  %s[0].organization  == "Twitter, Inc."
  %s[0].address       == "1355 Market Street\nSuite 900"
  %s[0].city          == "San Francisco"
  %s[0].zip           == "94103"
  %s[0].state         == "California"
  %s[0].country       == "United States"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4152229670"
  %s[0].fax           == "+1.4152220922"
  %s[0].email         == "domains-tech@twitter.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{6}
  %s[0] %CLASS{nameserver}
  %s[0].name == "d01-01.ns.twtrdns.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "d01-02.ns.twtrdns.net"
  %s[2] %CLASS{nameserver}
  %s[2].name == "r01-02.ns.twtrdns.net"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.p34.dynect.net"
  %s[4] %CLASS{nameserver}
  %s[4].name == "ns3.p34.dynect.net"
  %s[5] %CLASS{nameserver}
  %s[5].name == "r01-01.ns.twtrdns.net"
